nodejs/package.json

```diff
@@ -18,15 +18,15 @@
         "test": "mocha test/*.js",
         "buildProto": "pbjs -t static-module -w commonjs -o gtfs-realtime.js ../gtfs-realtime.proto"
     },
-    "engines": [
-        "node >= 0.10.0"
-    ],
+    "engines": {
+        "node": "18.12.1"
```

Comment:
Note that [the `engines` documentation uses ranges](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines) (e.g. `>= 18.12.1`). Not sure if the current *exact* version would cause a warning with Node `18.12.2`.

Reply:
`npm` seems to do the right thing with a specific version of `node` specified (still prints a warning when anyone attempts to run with a different version). since this can be granularly controlled via the docker image, i'd prefer to just keep this as specific as possible to avoid any possibility of drift, but happy to change this if anyone feels really strongly about it!